src/python/pants/option/global_options.py

```diff
@@ -201,3 +201,5 @@ def register_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--force', type=bool, recursive=True,
```

Comment:
It would make more code-sense to hang this off `CacheSetup`, but it would make no sense to a user (annoying as you say). We'd need the option to export a subsystem option into the global namespace I think. So, the current global `--force` is intentional with a command line use focus. Push back more if you want me to dig into this avenue.